x-pack/plugins/security_solution/public/resolver/view/process_event_dot.tsx

```diff
@@ -553,9 +559,7 @@ const UnstyledProcessEventDot = React.memo(
           >
             <EuiButton
               color={labelButtonFill}
-              data-test-subject="nodeLabel"
               fill={isLabelFilled}
-              id={labelId}
```

Comment:
same `id` and `data-test-subj` were being used for this button and its parent. the data-test-subj wasn't being used anyway.